identity-iota/src/tangle/encoding.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::tangle::compression_brotli::compress_brotli;
+use crate::tangle::compression_brotli::decompress_brotli;
+use crate::Error;
+
+#[derive(Copy, Clone)]
+pub(crate) enum MessageEncodingVersion {
+  JsonBrotli = 1,
+}
```

Comment:
Can we add an uncompressed/JSON-only encoding for debugging too?

```suggestion
pub(crate) enum MessageEncodingVersion {
  Json = 0,
  JsonBrotli = 1,
}
```